packages/agent/src/utils/oid4vciCredentialSuppliers.ts

```diff
@@ -226,9 +236,9 @@ const credentialDataSupplierSphereon: CredentialDataSupplier = (args: Credential
                     type: 'Sphereon Guest',
                 },
             },
-        } as unknown as CredentialDataSupplierResult)
-    }
-    throw Error(`${JSON.stringify(types)} not supported by this issuer`)
+        } as unknown as CredentialDataSupplierResult
+    } */
+    return Promise.resolve({} as unknown as CredentialDataSupplierResult) // return empty result so the code will start looking for templates
```

Comment:
Please resolve to undefined instead of an empty object

Reply:
I have tried but it's complicated, the function has to implement CredentialDataSupplier which is defined in dependency OID4VC
```
export type CredentialDataSupplier = (args: CredentialDataSupplierArgs) => Promise<CredentialDataSupplierResult>
```
It is not Promise<CredentialDataSupplierResult | undefined>

Reply:
Okay then create a TODO in the code here. We should move all the current hardcoded stuff to templates anyway, so this whole logic can go

Reply:
done